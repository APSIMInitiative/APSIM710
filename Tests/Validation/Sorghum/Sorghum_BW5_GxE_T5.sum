     ###     ######     #####   #   #     #   
    #   #    #     #   #        #   ##   ##   
   #     #   #     #   #        #   ##   ##   
   #######   ######     #####   #   # # # #   
   #     #   #              #   #   #  #  #   
   #     #   #         #####    #   #  #  #   
                                              
                                              
 The Agricultural Production Systems Simulator
             Copyright(c) APSRU               

Version                = 7.10
Title                  = Sorghum_BW5_GxE_T5
   Component                        "clock" = %apsim%\Model\Clock.dll
   Component                          "met" = %apsim%\Model\Input.dll
Paddock:
   Component                   "outputfile" = %apsim%\Model\Report.dll
   Component                        "accum" = %apsim%\Model\Accum.dll
   Component                   "operations" = %apsim%\Model\Operations.dll
   Component                   "fertiliser" = %apsim%\Model\Fertiliser.dll
   Component                   "Irrigation" = %apsim%\Model\Irrigation.dll
   Component  "Sow SkipRow on a fixed date" = %apsim%\Model\Manager.dll
   Component              "Harvesting rule" = %apsim%\Model\Manager.dll
   Component                 "IC_BW5 Water" = %apsim%\Model\SoilWat.dll
   Component              "IC_BW5 Nitrogen" = %apsim%\Model\SoilN.dll
   Component                      "sorghum" = %apsim%\Model\Sorghum.dll

------- clock Initialisation --------------------------------------------------
     Sequencer phases:
        prepare
        init_step
        do_management
        run_one_step
        do_pasture_water
        process
        do_pasture_growth
        do_stock
        do_cashbook
        post
        end_step
     Simulation start date = 26/09/1996
     Simulation end date   = 20/02/1997
     Time step =           = 1440 (mins)

------- met Initialisation ----------------------------------------------------
     Sparse data is not allowed
     INPUT File name: met\BW5_GxE.met

------- outputfile Initialisation ---------------------------------------------
     Output frequency:
        harvesting
     Output variables:
        today format dd/mm/yyyy as Date
        daysAfterSowing as Das
        day_of_year
        Stage
        Biomass
        GreenWt
        Biomass units g/m^2 as BiomassWt
        RootGreenWt
        LeafGreenWt
        StemGreenWt
        RachisGreenWt
        GrainGreenWt
        biomass_n
        GreenN
        RootGreenN
        LeafGreenN
        StemGreenN
        RachisGreenN
        GrainGreenN
        GrainGreenNConc
        GrainNo
        GrainSize
        HI
        LeafNo
        LAI
        SLAI
        ESW
        no3() as NO3
        SLN
        WaterSD
        sw()
        TPLA
        SPLA
        dlt_dm_green
        sw_supply
        radn_int
        DeltaGreenWt
        DeltaLAI
        DeltaGreenN
        DeltaLeafGreenN
        DeltaSLAI
        DeltaTT
        SenescedWt
        SenescedN
        RootSenescedN
        RootSenescedWt
        LeafSenescedWt
        LeafSenescedN
        RLV
        RootDepth
        RootProportion
        sum(sw_uptake)
        n_total_uptake
        n_supply_soil
        n_cum_uptake
        sw_demand
        sw_deficit
        root_depth
        no3
        sw
        sw_uptake
        n_sd_ratio
        no3_demand
        esw
        swdef_pheno
        swdef_photo
        swdef_expan
        nfact_pheno
        nfact_expan
        sw_dep
        n_diffusion_uptake
        n_massflow_uptake
        cover_green
        cover_tot
        DeltaTTFM
        st
        GrainTempFactor

     Output file = Sorghum_BW5_GxE_T5.out
     Format = normal

------- paddock.accum Initialisation ------------------------------------------
     Initialising

------- paddock.operations Initialisation -------------------------------------
     Operations Schedule
     ===================
      27/09/1996  Prepare      fertiliser apply amount = 10 (kg/ha), depth = 50 (mm), type = NO3_N ()
       9/10/1996  Prepare      fertiliser apply amount = 13 (kg/ha), depth = 50 (mm), type = NO3_N ()
      16/10/1996  Prepare      fertiliser apply amount = 99 (kg/ha), depth = 50 (mm), type = NO3_N ()

------- paddock.Irrigation Initialisation -------------------------------------
      Initialising
     
        - Reading Parameters
                      Irrigation parameters
          -----------------------------------------------
           Automatic Irrigation Application (Disabled)
           critical fraction of available soil water =  0.50
           depth for calculating available soil water =     600.00
           Irrigation Allocation Budget (Disabled)
          -----------------------------------------------

------- paddock.Sow SkipRow on a fixed date Initialisation --------------------
     Manager rules:
     
     SECTION:- start_of_day
     if (paddock_is_fallow() = 1 and today = date('10/10/1996')) then
       sorghum sow plants =10, sowing_depth = 30, cultivar = m35-1, row_spacing = 0.5, skip = solid, tiller_no_fertile = 0 ()
     endif
     END of rules
     

------- paddock.Harvesting rule Initialisation --------------------------------
     Manager rules:
     
     SECTION:- end_of_day
                if sorghum.stagename = 'harvest_ripe' or sorghum.plant_status = 'dead' then
       sorghum  harvest
       sorghum  end_crop
                endif
     END of rules
     

------- paddock.IC_BW5 Water Initialisation -----------------------------------
     
         - Reading constants
     
        - Reading Soil Property Parameters
     
        - Reading Soil Profile Parameters
     Initial soilwater distributed using "sw" parameter.
     
     
     
                      Soil Profile Properties
        ---------------------------------------------------------------------
              Depth  Air_Dry  LL15   Dul    Sat     Sw     BD   Runoff  SWCON
                mm     mm/mm  mm/mm  mm/mm  mm/mm  mm/mm  g/cc    wf
        ---------------------------------------------------------------------
            0.-  150.  0.100  0.210  0.420  0.440  0.281  1.320  0.762  0.160
          150.-  300.  0.210  0.220  0.390  0.410  0.338  1.490  0.190  0.110
          300.-  450.  0.230  0.240  0.360  0.380  0.353  1.540  0.048  0.140
          450.-  600.  0.230  0.240  0.360  0.380  0.360  1.540  0.000  0.140
          600.-  750.  0.220  0.230  0.390  0.410  0.390  1.490  0.000  0.110
          750.-  900.  0.220  0.230  0.420  0.440  0.407  1.470  0.000  0.110
          900.- 1050.  0.220  0.230  0.430  0.450  0.430  1.380  0.000  0.100
         1050.- 1200.  0.210  0.220  0.430  0.450  0.430  1.370  0.000  0.100
         1200.- 1350.  0.220  0.220  0.450  0.470  0.450  1.330  0.000  0.100
         1350.- 1500.  0.220  0.220  0.430  0.450  0.430  1.350  0.000  0.120
         1500.- 1650.  0.220  0.220  0.430  0.450  0.430  1.350  0.000  0.120
         1650.- 1800.  0.220  0.220  0.450  0.470  0.450  1.330  0.000  0.100
         1800.- 1950.  0.220  0.220  0.450  0.470  0.450  1.330  0.000  0.100
        ---------------------------------------------------------------------
     
     
     
                  Soil Water Holding Capacity
          ---------------------------------------------------------
              Depth    Unavailable Available  Max Avail.  Drainable
                          (LL15)   (SW-LL15)  (DUL-LL15)  (SAT-DUL)
                            mm        mm          mm         mm
          ---------------------------------------------------------
            0.-  150.      31.50      10.65      31.50       3.00
          150.-  300.      33.00      17.70      25.50       3.00
          300.-  450.      36.00      16.95      18.00       3.00
          450.-  600.      36.00      18.00      18.00       3.00
          600.-  750.      34.50      24.00      24.00       3.00
          750.-  900.      34.50      26.55      28.50       3.00
          900.- 1050.      34.50      30.00      30.00       3.00
         1050.- 1200.      33.00      31.50      31.50       3.00
         1200.- 1350.      33.00      34.50      34.50       3.00
         1350.- 1500.      33.00      31.50      31.50       3.00
         1500.- 1650.      33.00      31.50      31.50       3.00
         1650.- 1800.      33.00      34.50      34.50       3.00
         1800.- 1950.      33.00      34.50      34.50       3.00
          ---------------------------------------------------------
               Totals     438.00     341.85     373.50      39.00
          ---------------------------------------------------------
     
     
     
     
     
     
                  Initial Soil Parameters
          ---------------------------------------------------------
                 Insoil        Salb     Dif_Con   Dif_Slope
          ---------------------------------------------------------
                   0.00        0.11       40.00       16.00
          ---------------------------------------------------------
     
     
     
                  Runoff is predicted using scs curve number:
                Cn2  Cn_Red  Cn_Cov   H_Eff_Depth
                                           mm
          ---------------------------------------------------------
              85.00    0.80    0.20  450.00
          ---------------------------------------------------------
     
     
     
           Using Ritchie evaporation model
            Cuml evap (U):            6.00 (mm^0.5)
            CONA:                     3.50 ()
            Eo from priestly-taylor

------- paddock.IC_BW5 Nitrogen Initialisation --------------------------------
     
        - Reading Parameters
     
        - Reading Constants
     Using standard soil mineralisation for soil type BW5
     
     
     
           TAV and AMP supplied externally
     
     
     
                      Soil Profile Properties
          ------------------------------------------------
           Layer    pH    OC     NO3     NH4    Urea
                         (%) (kg/ha) (kg/ha) (kg/ha)
          ------------------------------------------------
             1     8.35  0.45   12.08    0.20    0.00
             2     8.52  0.32   12.52    0.22    0.00
             3     8.80  0.29    9.24    0.23    0.00
             4     8.95  0.23    7.62    0.23    0.00
             5     9.00  0.27    7.15    0.22    0.00
             6     9.00  0.30    8.16    0.22    0.00
             7     9.00  0.23   10.35    0.21    0.00
             8     9.00  0.22    6.16    0.21    0.00
             9     8.92  0.19    5.99    0.20    0.00
            10     8.97  0.12    4.05    0.20    0.00
            11     8.97  0.12    4.05    0.20    0.00
            12     8.82  0.14    3.99    0.20    0.00
            13     8.82  0.14    3.99    0.20    0.00
          ------------------------------------------------
           Totals               95.35    2.74    0.00
          ------------------------------------------------
     
     
     
     
     
     
                  Initial Soil Organic Matter Status
          ---------------------------------------------------------
           Layer      Hum-C   Hum-N  Biom-C  Biom-N   FOM-C   FOM-N
                    (kg/ha) (kg/ha) (kg/ha) (kg/ha) (kg/ha) (kg/ha)
          ---------------------------------------------------------
             1       8651.2   596.6   258.8    32.4    86.7     1.9
             2       7077.7   488.1    74.3     9.3    68.9     1.5
             3       6667.2   459.8    31.8     4.0    54.7     1.2
             4       5293.0   365.0    20.0     2.5    43.4     1.0
             5       6019.0   415.1    15.5     1.9    34.5     0.8
             6       6603.9   455.4    11.1     1.4    27.4     0.6
             7       4757.7   328.1     3.3     0.4    21.7     0.5
             8       4517.9   311.6     3.1     0.4    17.2     0.4
             9       3787.9   261.2     2.6     0.3    13.7     0.3
            10       2428.3   167.5     1.7     0.2    10.9     0.2
            11       2428.3   167.5     1.7     0.2     8.6     0.2
            12       2791.1   192.5     1.9     0.2     6.9     0.2
            13       2791.1   192.5     1.9     0.2     5.4     0.1
          ---------------------------------------------------------
           Totals   63814.1  4401.0   427.9    53.5   400.0     8.9
          ---------------------------------------------------------
------- Start of simulation  --------------------------------------------------
27 September 1996(Day of year=271), paddock.operations: 
      Sending apply message to fertiliser
27 September 1996(Day of year=271), fertiliser: 
     
   - Reading Fertiliser Type Parameters
        10.00 of N as nitrate (no3_n)
                                         added at depth   50.00 (layer   1)
9 October 1996(Day of year=283), paddock.operations: 
      Sending apply message to fertiliser
9 October 1996(Day of year=283), fertiliser: 
     
   - Reading Fertiliser Type Parameters
        13.00 of N as nitrate (no3_n)
                                         added at depth   50.00 (layer   1)
10 October 1996(Day of year=284), paddock.Sow SkipRow on a fixed date: 
     Manager sending message :- sorghum sow  plants = 10, sowing_depth = 30, cultivar = m35-1, row_spacing = 0.5, skip = solid, tiller_no_fertile = 0
10 October 1996(Day of year=284), sorghum: 
     Sowing initiate
     
                 Row spacing converted from m to mm
     
                 Crop Sowing Data
         -------------------------------------------------------
         Sowing   Depth  Plants Spacing Skiprow Cultivar    FTN
         Day no     mm     m^2    mm     code     name       no
         -------------------------------------------------------
            284    30.0    10.0   500    1.0     m35-1    0.00
         -------------------------------------------------------
     
      - reading constants for plant(sorghum) - m35-1
         -------------------------------------------------------
         tt_emerg_to_endjuv       =  252.00
         tt_flower_to_maturity    =  761.00
         tt_flag_to_flower        =  170.00
         tt_flower_to_start_grain =   80.00
         tt_maturity_to_ripe      =    1.00
         ------------------------------------------------------
     
     
                            Root Profile
         ---------------------------------------------------
              Layer       Kl           Lower    Exploration
              Depth     Factor         Limit      Factor
              (mm)         ()        (mm/mm)       (0-1)
         ---------------------------------------------------
             150.0     0.070          0.210       1.000
             150.0     0.070          0.220       1.000
             150.0     0.070          0.240       1.000
             150.0     0.070          0.240       1.000
             150.0     0.070          0.230       1.000
             150.0     0.070          0.230       1.000
             150.0     0.070          0.230       1.000
             150.0     0.070          0.220       1.000
             150.0     0.070          0.220       1.000
             150.0     0.040          0.220       1.000
             150.0     0.040          0.220       1.000
             150.0     0.040          0.220       1.000
             150.0     0.040          0.220       1.000
         ---------------------------------------------------
     
11 October 1996(Day of year=285), sorghum: 
     Stage 2 Germination  DAS = 1
     		Biomass       =   0.00 		 LAI            =   0.00
     		Stover N Conc =   0.00 		 Extractable SW =   4.04
13 October 1996(Day of year=287), sorghum: 
     Stage 3 Emergence  DAS = 3
     		Biomass       =   0.00 		 LAI            =   0.00
     		Stover N Conc =   0.00 		 Extractable SW =  10.74
16 October 1996(Day of year=290), paddock.operations: 
      Sending apply message to fertiliser
16 October 1996(Day of year=290), fertiliser: 
     
   - Reading Fertiliser Type Parameters
        99.00 of N as nitrate (no3_n)
                                         added at depth   50.00 (layer   1)
21 October 1996(Day of year=295), sorghum: 
     Stage 4 EndOfJuvenile  DAS = 11
     		Biomass       =   2.71 		 LAI            =   0.06
     		Stover N Conc =   0.11 		 Extractable SW =  47.60
11 November 1996(Day of year=316), sorghum: 
     Stage 5 FloralInitiation  DAS = 32
     		Biomass       =  76.44 		 LAI            =   1.35
     		Stover N Conc =   2.13 		 Extractable SW = 123.28
6 December 1996(Day of year=341), sorghum: 
     Stage 6 FlagLeaf  DAS = 57
     		Biomass       = 386.15 		 LAI            =   3.51
     		Stover N Conc =   7.39 		 Extractable SW = 192.56
22 December 1996(Day of year=357), sorghum: 
     Stage 7 Flowering  DAS = 73
     		Biomass       = 582.40 		 LAI            =   3.55
     		Stover N Conc =   8.89 		 Extractable SW = 244.82
28 December 1996(Day of year=363), sorghum: 
     Stage 8 StartGrainFill  DAS = 79
     		Biomass       = 673.65 		 LAI            =   3.55
     		Stover N Conc =   9.01 		 Extractable SW = 225.64
8 February 1997(Day of year=39), sorghum: 
     Stage 9 EndGrainFill  DAS = 121
     		Biomass       = 1150.30 		 LAI            =   2.09
     		Stover N Conc =   4.17 		 Extractable SW = 142.03
11 February 1997(Day of year=42), sorghum: 
     Stage 10 Maturity  DAS = 124
     		Biomass       = 1153.97 		 LAI            =   2.09
     		Stover N Conc =   4.11 		 Extractable SW = 141.05
12 February 1997(Day of year=43), sorghum: 
     Stage 11 HarvestRipe  DAS = 125
     		Biomass       = 1155.23 		 LAI            =   2.09
     		Stover N Conc =   4.11 		 Extractable SW = 140.73
12 February 1997(Day of year=43), paddock.Harvesting rule: 
     Manager sending message :- sorghum harvest
     
     Harvest
     Flowering (DAS)       = 73 		 Maturity (DAS)          = 124
     Flowering day         = 357 		 Maturity day            = 42
     Maximum LAI           = 3.6 		 Number of leaves        = 18.0
     Total above ground biomass    (kg/ha) = 11565.2
     Green above ground biomass    (kg/ha) = 11099.4
     Senesced above ground biomass (kg/ha) = 465.8
     Stover (kg/ha)        = 7690.2 	 Grain yield (kg/ha)     = 3875.0
     Grain % water content = 13.0 		 Grain yield wet (kg/ha) = 4454.1
     Weight 1000 grains(g) = 33.8 		 Grains/m^2              = 11478.5
     Grains/head           = 1147.8
     Grain N percent    (%)     =      1.26 	 Grain N            (kg/ha) =    49.00
     Total N content    (kg/ha) =     90.05 	 Senesced N content (kg/ha) =    15.12
     Green N content    (kg/ha) =     30.76
     
     Average Stress Indices:                          Water Photo  Water Expan  N Photo
        Emergence           to End of juvenile           1.0          1.0        0.9
        End of juvenile     to Floral initiation         1.0          1.0        1.0
        Floral initiation   to Flag leaf                 1.0          1.0        1.0
        Flag leaf           to Flowering                 1.0          1.0        1.0
        Flowering           to Start grain fill          1.0          1.0        1.0
        Start grain fill    to End grain fill            1.0          1.0        0.8
     
     Crop harvested.
        Organic matter removed from system:-      From Tops		From Roots
                         DM (kg/ha) =               3875.04		    0.00
                         N  (kg/ha) =                 49.00		    0.00
     Manager sending message :- sorghum end_crop
     Crop ended.
     Organic matter from crop:-      Tops to surface residue	 Roots to soil FOM
                         DM (kg/ha) =               7690.18		 2503.48
                         N  (kg/ha) =                 41.05		    4.83
                         P  (kg/ha) =                  0.00		    0.00
20 February 1997(Day of year=51), clock: 
     Simulation is terminating due to end criteria being met.

