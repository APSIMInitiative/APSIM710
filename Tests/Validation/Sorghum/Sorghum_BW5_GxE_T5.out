ApsimVersion = 7.10
precision = 8
Title = Sorghum_BW5_GxE_T5
           Date            Das    day_of_year          Stage        Biomass        GreenWt      BiomassWt    RootGreenWt    LeafGreenWt    StemGreenWt  RachisGreenWt   GrainGreenWt      biomass_n         GreenN     RootGreenN     LeafGreenN     StemGreenN   RachisGreenN    GrainGreenN GrainGreenNConc        GrainNo      GrainSize             HI         LeafNo            LAI           SLAI            ESW            NO3            SLN        WaterSD           sw()           TPLA           SPLA   dlt_dm_green      sw_supply       radn_int   DeltaGreenWt       DeltaLAI DeltaGreenN(1) DeltaGreenN(2) DeltaGreenN(3) DeltaGreenN(4) DeltaGreenN(5) DeltaLeafGreenN      DeltaSLAI        DeltaTT     SenescedWt      SenescedN  RootSenescedN RootSenescedWt LeafSenescedWt  LeafSenescedN         RLV(1)         RLV(2)         RLV(3)         RLV(4)         RLV(5)         RLV(6)         RLV(7)         RLV(8)         RLV(9)        RLV(10)        RLV(11)        RLV(12)        RLV(13)      RootDepth RootProportion(1) RootProportion(2) RootProportion(3) RootProportion(4) RootProportion(5) RootProportion(6) RootProportion(7) RootProportion(8) RootProportion(9) RootProportion(10) RootProportion(11) RootProportion(12) RootProportion(13) sum(sw_uptake) n_total_uptake  n_supply_soil   n_cum_uptake      sw_demand  sw_deficit(1)  sw_deficit(2)  sw_deficit(3)  sw_deficit(4)  sw_deficit(5)  sw_deficit(6)  sw_deficit(7)  sw_deficit(8)  sw_deficit(9) sw_deficit(10) sw_deficit(11) sw_deficit(12) sw_deficit(13)     root_depth         no3(1)         no3(2)         no3(3)         no3(4)         no3(5)         no3(6)         no3(7)         no3(8)         no3(9)        no3(10)        no3(11)        no3(12)        no3(13)          sw(1)          sw(2)          sw(3)          sw(4)          sw(5)          sw(6)          sw(7)          sw(8)          sw(9)         sw(10)         sw(11)         sw(12)         sw(13)   sw_uptake(1)   sw_uptake(2)   sw_uptake(3)   sw_uptake(4)   sw_uptake(5)   sw_uptake(6)   sw_uptake(7)   sw_uptake(8)   sw_uptake(9)  sw_uptake(10)  sw_uptake(11)  sw_uptake(12)  sw_uptake(13)     n_sd_ratio     no3_demand            esw    swdef_pheno    swdef_photo    swdef_expan    nfact_pheno    nfact_expan      sw_dep(1)      sw_dep(2)      sw_dep(3)      sw_dep(4)      sw_dep(5)      sw_dep(6)      sw_dep(7)      sw_dep(8)      sw_dep(9)     sw_dep(10)     sw_dep(11)     sw_dep(12)     sw_dep(13) n_diffusion_uptake n_massflow_uptake    cover_green      cover_tot      DeltaTTFM          st(1)          st(2)          st(3)          st(4)          st(5)          st(6)          st(7)          st(8)          st(9)         st(10)         st(11)         st(12)         st(13) GrainTempFactor
   (dd/mm/yyyy)         (days)             ()             ()        (kg/ha)        (g/m^2)        (g/m^2)        (g/m^2)        (g/m^2)        (g/m^2)        (g/m^2)        (g/m^2)        (g/m^2)        (g/m^2)        (g/m^2)        (g/m^2)        (g/m^2)        (g/m^2)        (g/m^2)             (%)   (grains/m^2)  (g/1000grain)             ()       (leaves)        (m2/m2)        (m2/m2)           (mm)        (kg/ha)      (g(N)/m2)             ()        (mm/mm)           (m2)           (m2)        (g/m^2)           (mm)             ()        (g/m^2)        (m2/m2)        (g/m^2)        (g/m^2)        (g/m^2)        (g/m^2)        (g/m^2)         (g/m^2)        (m2/m2)          (oCd)        (g/m^2)        (g/m^2)        (g/m^2)        (g/m^2)        (g/m^2)        (g/m^2)       (mm/mm3)       (mm/mm3)       (mm/mm3)       (mm/mm3)       (mm/mm3)       (mm/mm3)       (mm/mm3)       (mm/mm3)       (mm/mm3)       (mm/mm3)       (mm/mm3)       (mm/mm3)       (mm/mm3)           (mm)             (0-1)             (0-1)             (0-1)             (0-1)             (0-1)             (0-1)             (0-1)             (0-1)             (0-1)              (0-1)              (0-1)              (0-1)              (0-1)           (mm)        (g/m^2)        (g/m^2)        (g/m^2)           (mm)           (mm)           (mm)           (mm)           (mm)           (mm)           (mm)           (mm)           (mm)           (mm)           (mm)           (mm)           (mm)           (mm)           (mm)        (kg/ha)        (kg/ha)        (kg/ha)        (kg/ha)        (kg/ha)        (kg/ha)        (kg/ha)        (kg/ha)        (kg/ha)        (kg/ha)        (kg/ha)        (kg/ha)        (kg/ha)        (mm/mm)        (mm/mm)        (mm/mm)        (mm/mm)        (mm/mm)        (mm/mm)        (mm/mm)        (mm/mm)        (mm/mm)        (mm/mm)        (mm/mm)        (mm/mm)        (mm/mm)           (mm)           (mm)           (mm)           (mm)           (mm)           (mm)           (mm)           (mm)           (mm)           (mm)           (mm)           (mm)           (mm)             ()        (g/m^2)           (mm)             ()             ()             ()             ()             ()           (mm)           (mm)           (mm)           (mm)           (mm)           (mm)           (mm)           (mm)           (mm)           (mm)           (mm)           (mm)           (mm)            (g/m^2)           (g/m^2)             ()             ()          (oCd)           (oC)           (oC)           (oC)           (oC)           (oC)           (oC)           (oC)           (oC)           (oC)           (oC)           (oC)           (oC)           (oC)              ()
     12/02/1997            126             43    11.00000000 11951.01740800  1353.16506000  1195.10174000   213.35230300    58.11552700   546.58501000   129.14460300   405.96761700     9.00513700     8.20509800     0.41063100     0.87729200     1.36380600     0.38522300     5.16814500      1.27304400 11478.47015900    35.36774600     0.33969300    18.00886400     1.82021100     1.73168200   131.48167400   130.81222500     0.48197300     1.00000000     3.76319000  3551.89246300  1731.68178400     2.89596900     6.98557300    10.80878800     2.89596900     0.00000000     0.00000000     0.00000000     0.00000000     0.00000000     0.00000000      0.00000000     0.00000000     9.75000000    92.28452300     1.28284500     0.07217400    36.99553600    55.28898400     1.21067100     0.00000100     0.00477700     0.00647600     0.00761300     0.00769200     0.00769200     0.00769200     0.00769200     0.00769200     0.00769200     0.00769200     0.00769200     0.00769200  1950.00000000        1.00000000        1.00000000        1.00000000        1.00000000        1.00000000        1.00000000        1.00000000        1.00000000        1.00000000         1.00000000         1.00000000         1.00000000         1.00000000     0.73678900     0.00000000     0.00000000     9.45494300     0.73678900    36.50320800    21.51133000    14.18341800    13.51313000    17.78994800    21.25039300    21.72708900    21.18384900    22.27969700    14.83052400    12.87837200    14.69002500    13.94375200  1950.00000000    84.42619300     9.17452700     5.28494200     3.59636400     2.74778200     3.31795100     4.15428900     2.84454700     3.06717300     2.42349900     2.91717200     3.25202000     3.60575000     0.17664500     0.24639500     0.26525600     0.26969200     0.27109500     0.27797400     0.28474600     0.28826700     0.30086700     0.33066100     0.34362000     0.35150900     0.35646300     0.00000000     0.02944900     0.02817800     0.03312700     0.04584900     0.05352500     0.06108000     0.07616500     0.09022400     0.07032700     0.07856300     0.08357700     0.08672500     0.00000000     2.68556600   131.48167400     1.00000000     1.00000000     1.00000000     1.00000000     1.00000000    26.49678800    36.95921700    39.78840600    40.45374700    40.66420000    41.69607900    42.71183000    43.23998600    45.13007700    49.59914800    51.54306400    52.72639800    53.46952400         0.00000000        0.00000000     0.51716700     0.75846900    15.05000000    27.27998900    25.63869900    24.14325100    22.78043700    21.53806700    20.40490300    19.37061900    18.42573900    17.56158300    16.77022400    16.04442600    15.37761600    14.76381600      1.00000000
